### PR TITLE
docs: show complex object interactions in npm pkg

### PR DESCRIPTION
**Pull request by wraithgar**

### CODE REVIEW

File: docs/content/commands/npm-pkg.md [hunk 1]
```diff
@@ -76,6 +76,14 @@ Returned values are always in **json** format.
     npm pkg get contributors[0].email
     ```
 
+    For complex fields you can also use text indices in square brackets
+    to specifically select an child field. This is especially helpful
```
Comment (nlf) [inline docs/content/commands/npm-pkg.md]:
```suggestion
    to specifically select a child field. This is especially helpful
```

Comment (wraithgar) [reply docs/content/commands/npm-pkg.md]: That's what I get for mentally translating `an array` to `a child field` w/o proofreading.

**Review by nlf — APPROVED** [review]

File: docs/content/commands/npm-pkg.md [hunk 2]
```diff
@@ -76,6 +76,14 @@ Returned values are always in **json** format.
     npm pkg get contributors[0].email
     ```
 
+    For complex fields you can also use text indices in square brackets
+    to specifically select a child field. This is especially helpful
+    with the exports object:
+
+    ```bash
+    npm pkg get "exports[.].require"
```
Comment (ljharb) [inline docs/content/commands/npm-pkg.md:84]: "text indices" suggests like, 0, 1, 2, etc. is `.` special?

Comment (wraithgar) [reply docs/content/commands/npm-pkg.md:84]: No '.' is a valid ascii character hence "text"

Comment (ljharb) [reply docs/content/commands/npm-pkg.md:84]: right but "indices" are only numeric. if it's object properties then "properties" might be a better word?

Comment (wraithgar) [reply docs/content/commands/npm-pkg.md:84]:
```
    For complex fields you can also name a property in square brackets
    to specifically select a child field. This is especially helpful
    with the exports object:
```

Comment (ljharb) [reply docs/content/commands/npm-pkg.md:84]: Sounds great to me, thanks!

**Review by nlf — APPROVED** [review]

**Pull request merged**